catalyst_voices/packages/catalyst_voices_models/lib/src/wallet/wallet_header.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:catalyst_cardano/catalyst_cardano.dart';
+import 'package:catalyst_cardano_serialization/catalyst_cardano_serialization.dart';
+import 'package:catalyst_voices_models/src/wallet/wallet_metadata.dart';
+import 'package:equatable/equatable.dart';
+
+/// This class represents basic information's about Wallet,
+/// such as [CardanoWallet], but without specifying way to take action like
+/// enable.
+///
+/// Instance of this class can come from different places like backend
+/// or user interacting with wallet extension.
+final class WalletHeader extends Equatable {
```

Comment:
Header seems UI-ish, models should be clean of UI. What about calling it a WalletInfo, etc?

Reply:
I was thinking more like http headers but okey will think about smth else :)

Reply:
Changed to `WalletInfo`